<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Facultyportal</class>
 <widget class="QMainWindow" name="Facultyportal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Faculty Portal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>key.png</normaloff>key.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home Page</string>
     </attribute>
     <widget class="QLabel" name="WelcomeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>791</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome, &lt;name&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>791</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;This is the homepage of the Attendance management System&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Feel free to access through the tab system&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Time">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
      <property name="value" stdset="0">
       <double>941.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>941</number>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>427</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="gridVisible">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QCalendarWidget::NoSelection</enum>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::ShortDayNames</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Upload attendance</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>761</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QPushButton" name="AttendanceSubmit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>500</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit Attendance</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Check Attendance</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>611</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Check Class Attendance</string>
       </attribute>
       <widget class="QDateEdit" name="classAttendanceDate">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>50</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2023</year>
          <month>10</month>
          <day>25</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2030</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2020</year>
          <month>9</month>
          <day>1</day>
         </date>
        </property>
       </widget>
       <widget class="QPushButton" name="classAttendanceSubmit">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>50</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="classAttendanceShow">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>210</y>
          <width>351</width>
          <height>231</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.0597pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.0597pt; text-decoration: underline;&quot;&gt;StudentName	   Classes	Present&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.0597pt;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Check Student Attendance</string>
       </attribute>
       <widget class="QLineEdit" name="studentAttendanceUsername">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter Username:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="studentAttendanceSubmit">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="studentAttendanceShow">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>541</width>
          <height>391</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.0597pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.0597pt; text-decoration: underline;&quot;&gt;Date 	 Present(or)Absent&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.0597pt;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Send Notification</string>
     </attribute>
     <widget class="QLabel" name="usernametag">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>51</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(or)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NotificationSubmit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Notification</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UsernameSend">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>240</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="messagetag">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Message Here:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="MessageArea">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Send to All</string>
      </property>
     </widget>
     <widget class="QLabel" name="notificationAck">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>791</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Notifications</string>
     </attribute>
     <widget class="QPushButton" name="NotificationRefresh">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="notificationsWindow">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>671</width>
        <height>351</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.0597pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.0597pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NotificationClear">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
